Makefile

```diff
@@ -0,0 +1,24 @@
+#curBranch := $(shell git branch --show-current)
+curBranch := $(shell git rev-parse --abbrev-ref HEAD)
+export curBranch
+	
+list:
+	ls -l
+install:
+	echo Your current branch is: ${curBranch}	
+	git checkout utilities -- frameworks.zip lib.zip
```

Comment:
This is a cleaner approach, but still, I would not keep it in the repository but rather download it from the source when necessary.